### DISCUSSION TITLE
Integromat Integration

### DISCUSSION THREAD

**Discussion by viktaroa** [Budibase/budibase#1958] [Help] [answered]
I want to connect to integromat. The connection is ok but I need help in picking input values from the form fields as well as pushing results from integromat into the other fields in the same form.

So basically what I'm doing is creating a form where there are fields for locations, pickup and destination. When the form is submitted, it triggers a webhook to integromat, sending the locations. The aim is to send back the estimated distance and time from the results. From there it's displayed on another screen.

Currently I have been able to created the webhook connection. I did a test by manually typing values into the "Input Value" fields and it works fine when I check from integromat.

No my challenges are:
1. Automatically pick values from the form itself to fill the "Input Value" fields.
2. Send parts of the results (Distance & Duration) from integromat to update parts of the form field.

Any help please

**Comment by shogunpurple** [comment, marked as answer]
Hi @viktaroa!

### Step 1, binding your form values to integromat webhook
This requires setting up an "app action" automation and triggering it with a custom form.

https://www.loom.com/share/6e09f249cfef419192150cc24c199b90

### Step 2: Setting up integromat to hit budibase
I should caveat this by saying that you will not be able to update parts of the form field in your app live like you want, but you will be able to write the result to a table or somewhere else.

This will require some setup from your integromat scenario. You will need to get your integromat scenario to make an HTTP request to your budibase instance, and set up a webhook automation.

<img width="1648" alt="Screenshot 2021-07-09 at 10 51 49" src="https://user-images.githubusercontent.com/11256663/125059841-ab1c9400-e0a3-11eb-95fe-d5ea5bd60ad8.png">

You can find more information about webhooks here:
https://docs.budibase.com/automate/steps/triggers

**Reply by viktaroa** [reply]
Thanks a lot @shogunpurple will give it a try and send feedback.
